zetaclient/tx.go

```diff
@@ -51,6 +51,8 @@ func (b *ZetaCoreBridge) WrapMessageWithAuthz(msg sdk.Msg) (sdk.Msg, AuthZSigner
 }
 
 func (b *ZetaCoreBridge) PostGasPrice(chain common.Chain, gasPrice uint64, supply string, blockNum uint64) (string, error) {
+	// double the gas price to avoid gas price spike
+	gasPrice = gasPrice * 2
```

Comment:
We do it for every chains?
I think we should just keep it for Bitcoin once we fix the gas price increase mechanism.

Reply:
Yeah. The decision was yesterday to apply this hard coded multiplier `2` to every chain.

Reply:
Yeah, for this cherry-pick, this looks fine, but eventually, it might be an excellent idea to mode to chain params; we can always keep the values as 1 if we are not using it.

Reply:
This is a cherry-pick but we're also fixing the gas stability pool for Ethereum and for BSC the gas price is stable, this will cause unnecessary high fees. I think we can add a condition to only trigger for Bitcoin